src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java

```diff
@@ -83,57 +74,50 @@ public void parse_compulsoryFieldMissing_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
         // missing name prefix
-        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB, expectedMessage);
 
         // missing phone prefix
```

Comment:
change the comment to "// missing description prefix "

Reply:
done